charts/juicefs/templates/worker/statefuleset.yaml

```diff
@@ -129,6 +129,9 @@ spec:
             - name: cache-dir-{{ $name }}
```

Comment:
Please also update chart version and changelog. Thanks.

Reply:
done